### DISCUSSION TITLE
How to install on Nextjs15

### DISCUSSION THREAD

**Discussion by goodlythink** [mantinedev/mantine#7071] [Q&A]
How to install on Nextjs15

**Comment by goodlythink** [comment]
![image](https://github.com/user-attachments/assets/f5a34cd8-a4d0-40b2-8bf9-9faa40c405d7)
in mantine template can use Nextjs15 and React18
But default install  Nextjs15 it install React 19

**Comment by icflorescu** [comment]
Keep in mind that Next.js 15 with the app router will always use React 19, no matter what you're specifying in your `package.json`.
 
I'm currently building an app with Next.js 15 and only encountered a few minor issues so far (like [this one](https://github.com/mantinedev/mantine/issues/7028)). So you should be good to go, just 
Also, remember to `suppressHydrationWarning` for the `html` root tag).

**Reply by goodlythink** [reply]
Thanks a lot, bro! 😊
